Add Where the ISS at? API

ritikseeker opened the pull request:
1. Removed github jobs API
2. Updated playroomkit API
3. Added Where the ISS at? API

pratik-mahalle commented:
Hey @ritikseeker, can you please clarify why did you remove the github jokes API?

ritikseeker commented:
I removed GitHub jobs api, not github jokes api. I removed it because it no
longer works.

On Thu, 10 Jul, 2025, 4:45 pm Pratik Mahalle, ***@***.***>
wrote:

> *pratik-mahalle* left a comment (keploy/public-apis-collection#130)
> <https://github.com/keploy/public-apis-collection/pull/130#issuecomment-3057002752>
>
> Hey @ritikseeker <https://github.com/ritikseeker>, can you please clarify
> why did you remove the github jokes API?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/keploy/public-apis-collection/pull/130#issuecomment-3057002752>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXAXEIUWWPRKCNYRSLSTSKT3HZDN3AVCNFSM6AAAAAB76B3X4CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTANJXGAYDENZVGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

pratik-mahalle commented:
Cool, Please resolve the conflict !

pratik-mahalle reviewed (approved):
HI @ritikseeker 🎉 Big shoutout for your first PR here — welcome aboard!
Looking forward to more awesome work from you 👏
